src/types/flows.ts

```diff
@@ -135,6 +135,16 @@ export interface FlowList {
   }
 }
 
+export interface VariableMap {
+  [key: string]: Variable
+}
+
+// TODO: type this better. there are required properties, that have types, but
+// we also need this to stay open for panels to register whatever they want
+export interface QueryScope {
+  [props: string]: any
```

Comment:
nice. This might be one of those `type` vs `interface` things?

Reply:
man i hope someone figures this out. it was super confusing and googling anything about typescript just gives you "in typescript, === tests equality AND type"

Reply:
what happens if you create a general QueryScope interface that you then extend and add required properties to it?

Reply:
Tried that. Same error